src/RecipeCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const propTypes = {
+  recipe: PropTypes.object.isRequired,
```

Comment:
I would still validate the properties here 👇 
```suggestion
  recipe: PropTypes.shape({
    image: PropTypes.string.isRequired,
    title: PropTypes.string.isRequired,
    description: PropTypes.string.isRequired,
  }),
```